.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
   - TR_ARCH=x86_64
   - TR_ARCH=x86_64      CLANG=1
   - TR_ARCH=openj9-test
+  - TR_ARCH=vagrant-vdso
```

Comment:
I don't like the name for this job. vdso doean't mean that we run something without vdso. Does vagrant-fedodora-wo- vdso sound better?

Reply:
vagrant-fedora-no-vdso

Reply:
Renamed it to vagrant-fedora-no-vdso